kuma/wiki/views.py

```diff
@@ -1512,11 +1512,12 @@ def get_previous(revisions):
 @xframe_options_sameorigin
 @process_document_path
 @prevent_indexing
+@ratelimit(key='user_or_ip', rate='60/m', block=True)
```

Comment:
:+1: Do we want an even slower limit? `30/m` or even `15/m` seems like the limit that a human could do.